lib/docs/data_dictionary.md

```diff
@@ -100,3 +100,25 @@ The column name of the first column contains [embedded metadata](#embedded-metad
 | modifications | List of [modifications](#modifications) |
 | ppm | ppm tolerance |
 | rt_window | Window used for in-source decay correction (min) |
+
+### PGFinder Output
+
+| Field                             | Description                                                                                   |
+|:----------------------------------|:----------------------------------------------------------------------------------------------|
+| `Metadata`                        | Metadata about input files.                                                                   |
+| `ID`                              | Peak identifier.                                                                              |
+| `RT (min)`                        | Retention Time in minutes?                                                                    |
+| `Charge`                          | ?                                                                                             |
+| `Obs (Da)`                        | Observed mass in [Dalton (unit)](https://en.wikipedia.org/wiki/Dalton_(unit))                 |
+| `Theo (Da)`                       | Theoretical mass in  [Dalton (unit)](https://en.wikipedia.org/wiki/Dalton_(unit))             |
+| `Delta ppm`                       | Change in Parts Per Million?                                                                  |
```

Comment:
Yeah, the change being the difference between the observed and theoretical masses